packages/adapter/src/rest.ts

```diff
@@ -1276,7 +1276,7 @@ class RESTAdapter extends Adapter.extend(BuildURLMixin) {
         {
           status: `${status}`, // Set to a string per the JSON API spec: https://jsonapi.org/format/#errors
           title: 'The backend responded with an error',
-          detail: `${payload}`,
+          detail: JSON.stringify(payload),
```

Comment:
a comment explaining that `payload` should be a string already and that this is to provide a more meaningful string when payload is not a string but is not intended as public-api would be good.

Reply:
can do!

Reply:
done!